tests/test_integration/test_run_transfer/test_ftp.py

```diff
@@ -155,51 +164,44 @@ async def test_run_transfer_ftp_to_postgres(
     init_df: DataFrame,
     client: AsyncClient,
     ftp_to_postgres: Transfer,
+    ftp_file_connection: FTP,
     source_file_format,
     file_format_flavor,
+    tmp_path: Path,
 ):
-    # Arrange
     postgres, _ = prepare_postgres
     file_format, _ = source_file_format
+    file_format_in_path = "xlsx" if file_format == "excel" else file_format
 
-    # Act
-    result = await client.post(
-        "v1/runs",
-        headers={"Authorization": f"Bearer {group_owner.token}"},
-        json={"transfer_id": ftp_to_postgres.id},
+    await run_transfer_and_verify(client, group_owner, ftp_to_postgres.id)
+
+    reader = DBReader(
+        connection=postgres,
+        table="public.target_table",
     )
-    # Assert
-    assert result.status_code == 200
+    df = reader.run()
 
-    run_data = await get_run_on_end(
-        client=client,
-        run_id=result.json()["id"],
-        token=group_owner.token,
+    df, init_df = prepare_dataframes_for_comparison(df, init_df, file_format)
+    assert df.sort("id").collect() == init_df.sort("id").collect()
+    df_count = df.count()
+
+    add_increment_to_files_and_upload(
```

Comment:
My proposal is to split tests for full strategy and incremental strategy. In future, we may add other cases, like switching from incremental strategy to full, and this test will be huge.

For incremental strategy it's enough to test just one file format, as others are already tested in full strategy.